test/fuzzTesting/fuzzHelper.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file fuzzHelper.cpp
+ * @author Dimitry Khokhlov <winsvega@mail.ru>
+ * @date 2015
+ */
+
+#include "fuzzHelper.h"
+
+#include <chrono>
+#include <boost/random.hpp>
+#include <boost/filesystem/path.hpp>
+#include <libevmcore/Instruction.h>
+
+namespace dev
+{
+namespace test
+{
+
+boost::random::mt19937 RandomCode::gen;
+boostIntDistrib RandomCode::opCodeDist = boostIntDistrib (0, 255);
+boostIntDistrib RandomCode::opLengDist = boostIntDistrib (1, 32);
+boostIntDistrib RandomCode::uniIntDist = boostIntDistrib (0, 0x7fffffff);
+
+boostIntGenerator RandomCode::randOpCodeGen = boostIntGenerator(gen, opCodeDist);
+boostIntGenerator RandomCode::randOpLengGen = boostIntGenerator(gen, opLengDist);
+boostIntGenerator RandomCode::randUniIntGen = boostIntGenerator(gen, uniIntDist);
+
+std::string RandomCode::rndByteSequence(int length, SizeStrictness sizeType)
```

Comment:
All function parameters are prefixed with _ (e.g. _length). This needs to be done for all functions in this file.